Electron/RED/ControlCenter.tsx

```diff
@@ -2,27 +2,58 @@ import React, { Component } from "react"
 import { rovecomm } from "../Core/RoveProtocol/Rovecomm"
 import GPS from "./components/GPS"
 import Log from "./components/Log"
+import Map from "./components/Map"
+import Waypoints from "./components/Waypoints"
 
 interface IProps {}
 
-interface IState {}
+interface IState {
+  storedWaypoints: any
+  currentCoords: { lat: number; long: number }
+}
 
 class ControlCenter extends Component<IProps, IState> {
+  waypointsInstance: any
+
   constructor(props: any) {
     super(props)
-    this.state = {}
+    this.state = {
+      storedWaypoints: {},
+      currentCoords: { lat: 0, long: 0 },
```

Comment:
Add a button/keybind to center map view back on the rover?

Reply:
This is a good future feature consideration that will be considered later.